wandb/sdk/internal/sender.py

```diff
@@ -901,14 +922,15 @@ def send_run(self, record: "Record", file_dir: Optional[str] = None) -> None:
             config_value_dict = self._config_backend_dict()
             self._config_save(config_value_dict)
 
-        if is_wandb_init:
-            # Ensure we have a project to query for status
-            if run.project == "":
-                run.project = util.auto_project_name(self._settings.program)
-            # Only check resume status on `wandb.init`
-            error = self._maybe_setup_resume(run)
+        do_fork = self._settings.fork_from is not None and is_wandb_init
+        do_resume = bool(self._settings.resume)
 
-        if error is not None:
+        if do_fork and do_resume:
+            error = wandb_internal_pb2.ErrorInfo()
+            error.code = wandb_internal_pb2.ErrorInfo.ErrorCode.USAGE
+            error.message = (
+                "You cannot use `resume` and `fork_from` together. Please choose one."
+            )
             if record.control.req_resp or record.control.mailbox_slot:
```

Comment:
This should use the new `_handle_error()`, right?

Reply:
ah nice catch, fixed